src/en/reaperscans/src/eu/kanade/tachiyomi/extension/en/reaperscans/ReaperScans.kt

```diff
@@ -177,119 +101,71 @@ class ReaperScans : ParsedHttpSource() {
     // Details
     override fun mangaDetailsParse(document: Document): SManga {
```

Comment:
there is an endpoint for details too
`$apiUrl/series/$slug`
`https://api.reaperscans.com/series/ghost-story-club-remake`